airflow/jobs/scheduler_job_runner.py

```diff
@@ -1712,7 +1725,10 @@ def _find_zombies(self) -> None:
                 simple_task_instance=SimpleTaskInstance.from_ti(ti),
                 msg=str(zombie_message_details),
             )
-            self.log.error("Detected zombie job: %s", request)
+            # Refer: https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks
```

Comment:
I think the suggester meant to include this link in the log message so that the user could go visit the link and understand what Zombies are.

Reply:
Just move it to log message. I just tried to search the airflow codebase. It seems we use absolute links. I guess we might not have a better solution at this moment.